### ISSUE TITLE
Issue with symlinks where target is exactly 60 characters

### ISSUE THREAD

**Issue by ab3fx** [bestouff/genext2fs#19]
When a symlink target path is exactly 60 characters long, the final character is being truncated in the generated image.

For example, here is a symlink in my source directory:

/tmp/genimage/root/etc/ssl/certs/ca-cert-QuoVadis_Root_CA_3_G3.pem -> /usr/share/ca-certificates/mozilla/QuoVadis_Root_CA_3_G3.crt

Looking at the resulting image (mounted at /mnt), the final "t" is missing:

/mnt/etc/ssl/certs/ca-cert-QuoVadis_Root_CA_1_G3.pem -> /usr/share/ca-certificates/mozilla/QuoVadis_Root_CA_1_G3.cr

**Comment by bestouff** [comment]
Could you try to reproduce it on the master branch ? I tried here and it seems to work fine:
```
lrwxrwxrwx 1 xav  xav   60 juin  14 16:36 ca-cert-QuoVadis_Root_CA_3_G3.pem -> /usr/share/ca-certificates/mozilla/QuoVadis_Root_CA_3_G3.crt
lrwxrwxrwx 1 xav  xav   60 juin  14 16:35 l -> 012345678901234567890123456789012345678901234567890123456789
```

**Comment by josch** [comment]
@ab3fx is probably using version v1.4.1 of genext2fs. I can reproduce the problem there. The problem does not exist anymore in version v1.4.2. I tried to git bisect to find the commit where this problem was fixed but a lot of revisions between 1.4.1 and 1.4.2 fail to build because they cannot find cache.h.

Anyways, this problem existed in past versions of genext2fs but not anymore today. Hence this bug can be closed.

**Comment by josch** [comment]
I now found that `cache.h` and `list.h` from later revisions can be used. I found out that the following commit closes this issue: 0d5a40578666b3883f92e5863fe88a7f8ce7963f Looking at the content of this commit, this also makes sense because it fixes symlinks.

**Comment by bestouff** [comment]
Good job @josch (as usual).
Closing.
